ml-agents/mlagents/trainers/trainer_controller.py

```diff
@@ -254,8 +257,12 @@ def write_to_tensorboard(self, global_step: int) -> None:
                 trainer.write_summary(global_step, delta_train_start)
 
     def start_learning(
-        self, env_manager: SubprocessEnvManager, trainer_config: Dict[str, Any]
+        self, env_manager: EnvManager, trainer_config: Dict[str, Any]
     ) -> None:
+        """
+        Returns stats for each trainer.
```

Comment:
This isn't really a descriptive docstring for this function -- it really returns `None` but has a side effect of outputting trainer stats.  Probably should either change this or remove / push to a different PR.

Reply:
Actually I forgot to remove this - a previous attempt returned stats before Ervin pointed me to `_get_measure_vals`